opensearch/opensearch-kafka/postman/aiven-opensearch-products-demo.postman_collection.json

```diff
@@ -0,0 +1,288 @@
+{
+	"info": {
+		"_postman_id": "028a6aec-e29c-43b2-8ed7-abb263114694",
```

Comment:
is this associated with a real (your) postman account? perhaps remove it?

Reply:
this is the id of the postman collection, it changes every time on import

https://community.postman.com/t/while-importing-a-collection-file-to-postman-ids-are-newly-generated/16899